### PR TITLE
Full feature compatibility with WPF's Colors class

### PR DESCRIPTION
**Pull request by adrianknight89**
### Description of Change ###

WPF's `Colors` class has 141 colors from UNIX X11 color table. See https://msdn.microsoft.com/en-us/library/system.windows.media.colors(v=vs.110).aspx

XF had 19 colors, so I added the remaining for full compatibility.

**Questions**

- XF's definition of Pink is wrong. It should be `rgb(255, 192, 203)`. If we want to change it, it'll be a breaking change. How should this be handled? (Also XF Transparent is black with 0 alpha whereas .NET counterpart is white with 0 alpha. At least this doesn't represent a bug like that of Pink.)
- I added hex values of the colors since msdn shows hex in its own documentation as well as the color table. Should the original 19 colors also show hex? See `Color.cs`
- Original documentation is wrong in that it's using the wrong word RGB instead of HEX. For example: `White, the color that is represented by the RGB value #ffffff.` Should I fix this?

P.S. I automated most of the work, so there shouldn't be an error with the hex values.

### CODE REVIEW

**Comment by StephaneDelcroix** [comment]
before fixing the docs and everything, wait for @jassmith opinion on whether or not we will include this.

File: Xamarin.Forms.Core/Color.cs [hunk 1]
```diff
@@ -371,6 +371,130 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
 		public static readonly Color White = FromRgb(255, 255, 255);
 		public static readonly Color Yellow = FromRgb(255, 255, 0);
 
+		// remaining colors in WPF's System.Windows.Media.Colors
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Core/Color.cs]: but actually, I do not care

**Comment by adrianknight89** [comment]
I already provided the docs. GitHub won't show it because it's too big. You should be able to see it in your IDE.

**Comment by StephaneDelcroix** [comment]
my wrong, they are indeed at the bottom of the changes.

**Comment by StephaneDelcroix** [comment]
@adrianknight89 we'll take this change in, including the change in the Pink Color. Could you update the PR ?

thanks

File: Xamarin.Forms.Core/Color.cs [hunk 2]
```diff
@@ -371,6 +371,130 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
...
+		public static readonly Color SpringGreen = FromHex("#FF00FF7F");
+		public static readonly Color SteelBlue = FromHex("#FF4682B4");
+		public static readonly Color Tan = FromHex("#FFD2B48C");
+		public static readonly Color Thistle = FromHex("#FFD8BFD8");
+		public static readonly Color Tomato = FromHex("#FFFF6347");
+		public static readonly Color Turquoise = FromHex("#FF40E0D0");
+		public static readonly Color Violet = FromHex("#FFEE82EE");
+		public static readonly Color Wheat = FromHex("#FFF5DEB3");
+		public static readonly Color WhiteSmoke = FromHex("#FFF5F5F5");
+		public static readonly Color YellowGreen = FromHex("#FF9ACD32");
```
Comment (adrianknight89) [inline Xamarin.Forms.Core/Color.cs]: @StephaneDelcroix Should the docs be sorted as well?

Comment (StephaneDelcroix) [inline Xamarin.Forms.Core/Color.cs]: I don't think so. That's the responsibility of the displaying tool

File: Xamarin.Forms.Core/Color.cs [hunk 3]
```diff
@@ -371,6 +371,130 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
 		public static readonly Color White = FromRgb(255, 255, 255);
 		public static readonly Color Yellow = FromRgb(255, 255, 0);
 
+		// remaining colors in WPF's System.Windows.Media.Colors
+		public static readonly Color AliceBlue = FromHex("#FFF0F8FF");
```
Comment (adrianknight89) [inline Xamarin.Forms.Core/Color.cs]: Changed to `FromRgb()`.

File: Xamarin.Forms.Core/Color.cs (hunk 2 above)
Comment (adrianknight89) [inline Xamarin.Forms.Core/Color.cs]: Sorted colors alphabetically. Same was done in `ColorTypeConverter.cs`

**Comment by adrianknight89** [comment]
I updated Pink to reflect the correct value. I also updated Transparent to be transparent from white instead of black. The end result should be the same since the alpha is 0.

Should we remove the obsolete color? It was set to obsolete in 1.3. I think it's time to get rid of it and let devs update to the correct spelling.

**Comment by StephaneDelcroix** [comment]
@adrianknight89 unfortunately we can't remove API, even the ones Obsoleted for a long time.

**Pull request merged**